src/EPROMStore.cpp

```diff
@@ -399,23 +409,32 @@ float EEPROMStore::getRAStepsPerDegree()
 // Store the RA steps per degree (actually microsteps per degree).
 void EEPROMStore::storeRAStepsPerDegree(float raStepsPerDegree)
 {
-    int32_t val = raStepsPerDegree * 10;  // Store as tenths of degree
-    val         = clamp(val, (int32_t) INT16_MIN, (int32_t) INT16_MAX);
+    // Store steps as 100x steps/deg at 256 MS.
+    float factor = 25600.0 / RA_TRACKING_MICROSTEPPING;
+    int32_t val  = raStepsPerDegree * factor;
     LOG(DEBUG_EEPROM, "[EEPROM]: Storing RA steps to %d (%f)", val, raStepsPerDegree);
 
-    updateInt16(RA_STEPS_DEGREE_ADDR, val);
-    updateFlags(RA_STEPS_FLAG);
+    updateInt32(RA_NORM_STEPS_DEGREE_ADDR, val);
+    updateFlagsExtended(RA_NORM_STEPS_MARKER_FLAG);
     commit();  // Complete the transaction
 }
 
-// Return the DEC steps per degree (actually microsteps per degree).
+// Return the DEC steps per degree for guiding (actually microsteps per degree).
 // If it is not present then the default uncalibrated DEC_STEPS_PER_DEGREE value is returned.
 float EEPROMStore::getDECStepsPerDegree()
 {
     float decStepsPerDegree(DEC_STEPS_PER_DEGREE);  // Default value
 
-    if (isPresent(DEC_STEPS_FLAG))
+    if (isPresentExtended(DEC_NORM_STEPS_MARKER_FLAG))
+    {
+        // This version stored 100x steps/deg for 256 MS
+        float factor      = 25600.0 / DEC_GUIDE_MICROSTEPPING;
```

Comment:
Did you test this with different `*_MICROSTEPPING` `#define`s? Do you need to clear EEPROM if you change that stuff, cause the math gets out of sync with the EEPROM?

Reply:
Yes, I flashed my board with different MS settings. The whole point of this PR is to NOT have to reset the EEPROM when changing MS settings....